t/api/04-jobs.t

```diff
@@ -1128,19 +1117,34 @@ subtest 'Parse extra tests results - LTP' => sub {
       or die diag explain path($basedir)->list_tree;
     ok -e path($basedir, 'LTP-LTP_syscalls_accept01.txt'), 'LTP was parsed';
 
-    # Now we check what parser expects to have (this have been generated from openQA side)
     $parser->results->each(
         sub {
-            my $db_module = $t->app->schema->resultset('Jobs')->find(99963)->modules->find({name => $_->test->name});
-
-            ok -e path($basedir, 'details-' . $_->test->name . '.json'),
-              'detail from junit was written for ' . $_->test->name;
-            is_deeply $db_module->results->{details}, $_->details;
-            is $db_module->name, $_->test->name, 'Modules name are matching';
-            is $db_module->script, 'test', 'Modules script are matching';
-            is $db_module->category, $_->test->category, 'Modules category are matching';
-            is $db_module->result, ($_->result eq 'ok' ? 'passed' : 'failed'), 'Modules can be passed or failed';
-            ok -e path($basedir, $_->{text}) for @{$db_module->results->{details}};
+            my $testname = $_->test->name;
+            subtest "Parsed results for $testname" => sub {
+                my $db_module = $t->app->schema->resultset('Jobs')->find(99963)->modules->find({name => $testname});
+
+                ok(-e path($basedir, "details-$testname.json"), 'junit details written');
+                my $got_details = {
+                    results => {
+                        details => $db_module->results->{details},
```

Comment:
IMHO the level of indention has grown too much if you look at this example.

Reply:
I'm cosolidating the checks because hundreds of lines of logs are not readable.

Reply:
Yes, I appreciate that but making the code less readable is a questionable drawback :)